spec/features/create_invoice_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
```

Comment:
@yunixon also add a basic spec for Edit and Update Invoice and for Delete Invoice.